### PR TITLE
C22_WMDE_Test_EN_07

### PR DESCRIPTION
**Pull request by Abban**
* Uses CTRL from test 06
* VAR uses the new "I've already donated" component

Ticket: https://phabricator.wikimedia.org/T325149

### CODE REVIEW

**Review by gbirke — APPROVED** [review]

File: banners/english/banner_var.js [hunk 1]
```diff
@@ -52,7 +54,8 @@ bannerPresenter.present(
 		footer: Footer,
 		bannerText: BannerText,
 		slides: Slides,
-		translations: Object.assign( Translations, LocalTranslations, TranslationsSoftClose ),
+		alreadyDonatedContent: AlreadyDonated,
+		translations: Object.assign( Translations, TranslationsAlreadyDonated, TranslationsSoftClose, LocalTranslations ),
 		formItems: createFormItems( Translations, formatters.amountInputFormatter ),
 		bannerType: BannerType.VAR,
 		showCookieBanner: '0',
```
Comment (moiikana) [inline banners/english/banner_var.js]: could get thrown out at this or another point

**Review by moiikana — APPROVED** [review]

File: banners/english/banner_var.js (hunk 1 above)
Comment (Abban) [reply banners/english/banner_var.js]: I'm not sure what you mean?

Comment (moiikana) [reply banners/english/banner_var.js]: oh, I thought I highlighted this line, that was the only thing I meant to talk about:

showCookieBanner: '0',

Comment (Abban) [reply banners/english/banner_var.js]: Ah I see, the highlights make the display weird. Removed

**Pull request merged**